<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageWidget</class>
 <widget class="QWidget" name="PageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelTotalCount">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ComboLine" name="lineMaxRows">
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>行</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPre">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>上一页</string>
     </property>
     <property name="text">
      <string>＜</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ComboLine" name="lineCurrentPage">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPageCount">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnNext">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>下一页</string>
     </property>
     <property name="text">
      <string>＞</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboLine</class>
   <extends>QLineEdit</extends>
   <header>src/fend/uicommon/comboline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
